docker/chronicle.yaml

```diff
@@ -64,21 +66,26 @@ services:
 
   chronicle-sawtooth-tp:
     environment: &rustenv
-      - &rustbacktrace RUST_BACKTRACE=full
-      - &rustlog RUST_LOG=debug,cranelift_codegen=off,wasmtime_cranelift=off,regalloc2::ion=off,cranelift_wasm=off
+      - RUST_BACKTRACE=full
+      - RUST_LOG=info,cranelift_codegen=off,wasmtime_cranelift=off,regalloc2::ion=off,cranelift_wasm=off
     image: &chronicleTpImage chronicle-tp-amd64:${ISOLATION_ID:-local}
-    entrypoint: >
-      /bin/bash -xc "/usr/local/bin/chronicle_sawtooth_tp
-      -C tcp://validator:4004
-      --console-logging pretty"
+    entrypoint: /bin/bash
+    command:
+      - -exc
+      - |
+        sleep 40
```

Comment:
This one shouldn't really be necessary - if the validator isn't up the usual TP approach is to keep looping and retrying until it comes up